backend/src/Manager/User/UserManager.php

```diff
@@ -19,7 +20,7 @@ class UserManager
     private UserPasswordHasherInterface $encoder;
     private UserEntityRepository $userRepository;
 
-    public function __construct(AutoMapping $autoMapping, EntityManagerInterface $entityManager, UserPasswordHasherInterface $encoder, UserEntityRepository $userRepository)
+    public function __construct(AutoMapping $autoMapping, EntityManagerInterface $entityManager, UserPasswordHasherInterface $encoder, UserEntityRepository $userRepository, private ChatRoomManager $chatRoomManager)
```

Comment:
We need to define chatRoomManager as a class parameter, then initialize it in the constructor.